common/shared/src/main/java/app/coronawarn/server/common/shared/exception/UnableToLoadFileException.java

```diff
@@ -0,0 +1,17 @@
+
+
+package app.coronawarn.server.common.shared.exception;
+
+/**
+ * The file could not be loaded/parsed correctly.
+ */
+public class UnableToLoadFileException extends Exception {
```

Comment:
In the distribution service this class exists too ( and is actually used)

Reply:
Yes, indeed. I removed it from distribution and referenced it from shared.